src/hooks/users.ts

```diff
@@ -28,7 +110,7 @@ const useConnectVenueUsers = () => {
 export const useVenueUsers = (): readonly WithId<User>[] => {
   useConnectVenueUsers();
 
-  return useSelector(usersSelector) ?? [];
+  return useSelector(universeUsersSelector) ?? [];
```

Comment:
Since this is in`useVenueUsers` it should probably use a selector called `venueUsersSelector`. Alternatively, the name of this hook should be changed to something like `useUniverseUsers`.